src/main/java/com/google/sps/data/Event.java

```diff
@@ -48,119 +45,54 @@
    * 
    * @param name name of the location (NOT address)
    * @param address exact address of the POI
-   * @param date date from user input travel day. Cannot be null.
-   * @param startTime start of activity (HHMM format)
+   * @param startTime start of activity (YYYY-MM-DD'T'HH:MM:SS)
    * @param travelTime time spent traveling to next location (minutes). 
    *        Null if last location of the day.
    * @param timeAtLocation time spent at POI (minutes)
    */
-  public Event(String name, String address, String date, int startTime, 
+  public Event(String name, String address, LocalDateTime startTime, 
               int travelTime, int timeAtLocation) {
```

Comment:
Is there a reason why `timeAtLocation` is an `int`, which is then converted to a `long`? Could it simply initially be a `long`? (That being said, it's a simple switch, so no big deal if you want to keep this structure.)

Reply:
I put it as an int that way theres no error when creating the event since I will probably get an int from Eshika (Long to int conversions sometimes throws errors).